lib/node/node.ts

```diff
@@ -83,3 +85,28 @@ if (httpClient && httpClient.ClientRequest) {
     }
   };
 }
+
+function patchNextTick() {
+  let setNative = null;
+
+  function scheduleTask(task: Task) {
+    const args = task.data;
+    args[0] = function() {
+      task.invoke.apply(this, arguments);
+    };
+    setNative.apply(process, args);
```

Comment:
Do we know it's `process`?

```js
var nextTick = process.nextTick;
nextTick.call(myObject, function() {
  console.log(myObject); // If nextTick didn't always bind to `global`,
                         // this would be `myObject`.  However, with the 
                         // zone.js patch, this would be `process`
});
```